phylanx/util/matrix_iterators.hpp

```diff
@@ -7,6 +7,7 @@
 #if !defined(PHYLANX_MATRIX_ITERATORS)
 #define PHYLANX_MATRIX_ITERATORS
 
+#include <algorithm>
```

Comment:
What do you need this #include for?

Reply:
I don't need it. Inspect was complaining.